openhands/agenthub/codeact_agent/prompts/additional_info.j2

```diff
@@ -24,5 +24,8 @@ For example, if you are using vite.config.js, you should set server.host and ser
 {% if runtime_info.additional_agent_instructions %}
 {{ runtime_info.additional_agent_instructions }}
 {% endif %}
+{% if runtime_info.date or runtime_info.utc_time %}
+Today's datetime is {{ runtime_info.date }} at {{ runtime_info.utc_time }} (UTC).
```

Comment:
huh, shall we just keep the date then? 

If the agent need specific time, it can execute the command to figure it out?

Reply:
Yeah sounds good!

Reply:
I'll open a separate PR to include timestamps for PS1 bash commands